platform/src/test/java/platformstepdefinition/PF_Footer.java

```diff
@@ -0,0 +1,51 @@
+package platformstepdefinition;
+
+import org.junit.runner.RunWith;
+import org.openqa.selenium.WebDriver;
+
+import base.NewBaseClass;
+import cucumber.api.PendingException;
+import cucumber.api.java.en.And;
+import cucumber.api.java.en.Given;
+import cucumber.api.java.en.Then;
+import cucumber.api.junit.Cucumber;
+import platform.pageobjects.Authentication.LoginPage;
+import platform.pageobjects.Footer.FooterPage;
+
+@RunWith(Cucumber.class)
+public class PF_Footer extends NewBaseClass {
+
+	public WebDriver driver;
+
+	@Given("^Navigate to URL and check footer links (.+)$")
+	public void navigate_to_URL_and_check_footer_links(String browser) throws Throwable {
+		driver = openbrowser(browser);
+		FooterPage footer = new FooterPage(driver);
+		footer.footerFunctions();
+	}
+
+	@And("^check footer links with after login (.+)$")
+	public void check_footer_links_after_login(String browser) throws Throwable {
+		LoginPage loginToPlatform = new LoginPage(driver);
+		loginToPlatform.loginToPlatform();
+		FooterPage footer = new FooterPage(driver);
+		footer.afterLoginFooter();
```

Comment:
Rename to verb-non format
Cant determine what this method does

Reply:
Done changes as requested.

Reply:
Thanks